### PR TITLE
Add project capability for inproc hosting

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.targets [hunk 1]
```diff
@@ -11,4 +11,12 @@
     -->
     <AspNetCoreModuleHostingModel Condition=" '$(AspNetCoreModuleHostingModel)' == '' AND '$(DisableInProcessHosting)' != 'true' ">InProcess</AspNetCoreModuleHostingModel>
   </PropertyGroup>
+  
+  <!--
+    Capability that enables Visual Studio supoort for hosting Asp.Net Core applications in the IIS process
```
Comment (natemcmaster) [inline src/Microsoft.AspNetCore.Server.IIS/Microsoft.AspNetCore.Server.IIS.targets]: typo: support

**Comment by jkotalik** [comment]
Do we still need both of these properties? Otherwise LGTM

**Comment by shirhatti** [comment]
Capability is to indicate whether in-process is supported or not, i.e., should tooling even give you the option to switch.
Property is to indicate what mode we're currently operating in.

**Comment by jkotalik** [comment]
For #532

**Comment by shirhatti** [comment]
@jkotalik Can you approve?

**Comment by shirhatti** [comment]
> Property is to indicate what mode we're currently operating in.

Based on this and our offline conversation we should no longer set the `AspNetCoreModuleHostingModel` property

**Pull request merged**